Updated hyperlink topic (#398)
IgniteUI/indigo-design-docfx: en/components/hyperlink.md

## Size

The Hyperlink comes in two different sizes matching the available sizes for paragraph text: Body 1 and Body 2.

<img class="responsive-img" src="../images/hyperlink_sizes.png" srcset="../images/hyperlink_sizes@2x.png 2x" />

## Url

Specify the link you want the hyperlink to open when clicked via the Url override. This will have no visibility in your design but will be taken into account by the code generation services.

## Styling

> The Hyperlink can be styled through the Styling library by changing its default blue color to a different one.

Reviewer: ```suggestion
The Hyperlink can be styled by changing the default blue color of the text of the `Symbol Master` that can be found on the 🧩 Components page in the Indigo.Design library.
```

> 

Reviewer: ```suggestion

## Url

Specify the link you want the hyperlink to open when clicked via the Url override. This will have no visibility in your design but will be taken into account by the code generation services.

```

## Styling

> The Hyperlink can be styled through the Styling library by changing its default blue color to another one, and you can change the URL and text.

Reviewer: ```suggestion
The Hyperlink can be styled through the Styling library by changing its default blue color to a different one.
```
